src/index.js

```diff
@@ -20,4 +21,4 @@ const styled = _styled(_styledComponent(_ComponentStyle(generateAlphabeticName))
 
 /* Export everything */
 export default styled
-export { css, keyframes, injectGlobal, ThemeProvider }
+export { css, keyframes, injectGlobal, ThemeProvider, ServerStyleSheet }
```

Comment:
You're exporting a singleton instance, so I'd go with a name like `serverStyles`